lib/pharos/host/ubuntu/scripts/configure-cri-o.sh

```diff
@@ -62,11 +62,9 @@ fi
 
 export DEBIAN_FRONTEND=noninteractive
 apt-mark unhold cri-o
-apt-get install -y cri-o="${CRIO_VERSION}"
+apt-get install -y --allow-downgrades cri-o="${CRIO_VERSION}"
```

Comment:
I guess we need this flag also on other distros

Reply:
Fixed.